teuthology/provision/test/test_fog.py

```diff
@@ -291,7 +291,7 @@ def test_wait_for_ready(self, tries):
         connect_results = [MaxWhileTries for i in range(tries)] + [True]
         obj = self.klass('name.fqdn', 'type', '1.0')
         self.mocks['m_Remote_connect'].side_effect = connect_results
-        if tries >= 50:
+        if tries >= 100:
```

Comment:
Change the 51 to 101 above

Reply:
It passed with it as 100??

https://jenkins.ceph.com/job/teuthology-pull-requests/1131/console

Reply:
@zmc Do you mean this one? https://github.com/ceph/teuthology/blob/f987311acfadb4d40147e33472d1b575c0d449e3/teuthology/provision/test/test_fog.py#L288

Reply:
yep.  that construct runs that function once with 'tries == 1' and once with 'tries == 101'

Reply:
@djgalloway Yes, thanks!